### PR TITLE
Created sign up form

### PR DESCRIPTION
**Pull request by brookewoolley**
Created sign up form in users/new view

### CODE REVIEW

File: app/views/sessions/mocklogin.html.erb [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
```
Comment (thielsen) [inline app/views/sessions/mocklogin.html.erb:1]: Do we need this file?

File: config/routes.rb [hunk 2]
```diff
@@ -8,5 +8,5 @@
 
   resources :spaces, :users
 
-  root 'sessions#new'
+  root 'welcome#index'
```
Comment (thielsen) [inline config/routes.rb:11]: Should point to users#new - I'll approve and fix..

File: coverage/.resultset.json [hunk 3]
```diff
@@ -1,26 +1,63 @@
 {
```
Comment (thielsen) [inline coverage/.resultset.json:1]: Will add this to .gitignore

**Review by thielsen — APPROVED** [review]
A few points here but will approve and fix in my upcoming pull

File: app/views/sessions/mocklogin.html.erb (hunk 1 above)
Comment (david-mears) [reply app/views/sessions/mocklogin.html.erb:1]: Yes, it's the styling for the log in page which I spent yesterday doing.

**Review by aodhan-dev — APPROVED** [review]

**Pull request merged**